CMakeLists.txt

```diff
@@ -220,6 +219,19 @@ target_compile_definitions      (libtesseract
 )
 set_target_properties           (libtesseract PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS True)
 endif()
+if (Tesseract_USE_OPENCL)
+    target_compile_definitions  (libtesseract PRIVATE USE_OPENCL=1)
+    if (APPLE OR ANDROID)
+        # Inbuilt to NDK
+        find_package(OpenCL REQUIRED)
+        target_link_libraries       (libtesseract ${OpenCL_LIBRARIES})
+        target_include_directories  (libtesseract PRIVATE ${OpenCL_INCLUDE_DIRS})
+    else()
+        hunter_add_package(OpenCL)
+        find_package(OpenCL CONFIG REQUIRED)
+        target_link_libraries       (libtesseract OpenCL::OpenCL)
```

Comment:
I see clFinish in list of undefined symbols of installed library, meaning dependency is public.